modules/installation-osp-config-yaml.adoc

```diff
@@ -49,3 +48,57 @@ endif::openshift-origin[]
 pullSecret: '{"auths": ...}'
 sshKey: ssh-ed25519 AAAA...
 ----
+
+.Example dual stack `install-config.yaml` file
```

Comment:
Maybe not long enough to warrant, but possible candidate for `[%collapsible]`, as well as the "single stack" example YAML above.